test/test_model.py

```diff
@@ -813,6 +813,8 @@ def test_function_set_error_legacy(self):
         calls = [["action-set", "foo=bar", "dead=beef cafe"]]
         self.assertEqual(fake_script_calls(self, clear=True), calls)
 
+        self.assertEqual(fake_script_calls(self, clear=True), [])
+
```

Comment:
I'm not sure how the merge resolution seems to have added this.  It's in the original commit for this PR, untouched, it's not in master, but I also don't see any commit that removed it.  I'm going to remove it anyway, since it's not supposed to be a part of this PR and we can address it later if needed.